web/src/components/card.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Card() {
+    return (
+        <div className="absolute bg-white mx-4 md:ml-10 rounded-bl-lg w-11/12 md:w-5/12">
+            {/* left-24 rounded-bl-lg top-44 w-5/12 */}
+            <h1 className="header-accent pl-6 pt-7 mb-6 text-brand-1 text-4xl uppercase">
+                Hello
+            </h1>
+            <p className="mt-6 pl-6 pb-7">
```

Comment:
Can we combine the bottom margin from the heading and the top margin from the paragraph? I like to stick to `margin-bottom` for spacing elements vertically whenever possible. 

And sorry to nitpick a class name but `heading-accent` is probably a better name than `header-accent`